README.md

```diff
@@ -1,25 +1,211 @@
 [![codecov](https://codecov.io/gh/chanzuckerberg/cell-census/branch/main/graph/badge.svg?token=byX1pyDlc9)](https://codecov.io/gh/chanzuckerberg/cell-census)
 
-# CELLxGENE Cell Census
+# Cell Census of CZ CELLxGENE Discover
 
-**Status**: Unstable, under rapid development
+**CZ CELLxGENE Discover** ([cellxgene.cziscience.com](https://cellxgene.cziscience.com/)) is a free-to-use data portal hosting a growing corpus of more than **700 single-cell datasets** comprising about **50 million cells** from the major human and mouse tissues. The portal provides a set of visual tools to download and explore the data. **All data is standardized** to include raw counts and a common vocabulary for gene and cell metadata.
```

Comment:
In general, it's helpful to include line breaks so that reviewers can comment on a specific line rather than a paragraph. 


`**CZ CELLxGENE Discover** ([cellxgene.cziscience.com](https://cellxgene.cziscience.com/))
`
suggest:

`[**CZ CELLxGENE Discover**](https://cellxgene.cziscience.com/)
`

`**All data is standardized**
`
suggest:

`**All data is [standardized](https://github.com/chanzuckerberg/single-cell-curation/tree/main/schema/3.0.0)**`